script/deploy/facets/utils/ScriptBase.sol

```diff
@@ -28,14 +28,29 @@ contract ScriptBase is Script, DSTest {
     // reads an address from a config file and makes sure that the address contains code
     function _getConfigContractAddress(
         string memory path,
-        string memory key
+        string memory key,
+        bool allowZeroAddress // if zeroAddress is found, it will not check for contract and stil return the address
```

Comment:
maybe instead of having false everywhere for allowZeroAddress we could use overloading and treat allowZeroAddress as optional. Like:

```
function _getConfigContractAddress(
    string memory path,
    string memory key
) internal returns (address contractAddress) {
    return _getConfigContractAddress(path, key, false); // where _getConfigContractAddress with three params is with allowZeroAddress
}
```

wdyt?

Reply:
good idea but it's already done and it's a tedious PR for a simple helper function...if you feel keen please could you implement and push the change or let's just move on?

Reply:
that's true, let me take care of it

Reply:
changed